### PR TITLE
Add mapping for data sources

### PR DESCRIPTION
**Pull request by kononenkoAnton**
## Description

[Monday ticket](add your monday ticket url here)

`Please describe PR here`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/quick-brick-login-flow/src/Services/localStorageService.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const tokenKey = "token";
 const responseKey = "data";
 const idTokenKey = "id_token";
 const refreshUrl = "refresh_url";
-const emailKey = "email";
+const emailKey = "user_id";
```
Comment (ranmeirman) [inline plugins/quick-brick-login-flow/src/Services/localStorageService.ts:9]: Why not username?

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: plugins/quick-brick-login-flow/src/Services/localStorageService.ts (hunk 1 above)
Comment (kononenkoAnton) [reply plugins/quick-brick-login-flow/src/Services/localStorageService.ts:9]: because it can be email as well

Comment (kononenkoAnton) [reply plugins/quick-brick-login-flow/src/Services/localStorageService.ts:9]: @ranmeirman do you want me to change to username?\

Comment (ranmeirman) [reply plugins/quick-brick-login-flow/src/Services/localStorageService.ts:9]: I think we should, no?

Comment (kononenkoAnton) [reply plugins/quick-brick-login-flow/src/Services/localStorageService.ts:9]: ok, will change. I think about something more general

**Pull request merged**